src/ui/playblast_ui.py

```diff
@@ -41,12 +71,23 @@ def __init__(self, parent=None):
 
         # Second row
         second_row = QLabel("Frame rate:")
-        self.rate_line_edit = QLineEdit()
+        self.rate_line_edit = QComboBox()
+        self.rate_line_edit.addItem("24")
+        self.rate_line_edit.addItem("23.976")
+        self.rate_line_edit.addItem("25")
+        self.rate_line_edit.addItem("29.97")
+        self.rate_line_edit.addItem("30")
+        self.rate_line_edit.addItem("40")
+        self.rate_line_edit.addItem("60")
```

Comment:
Me gusta la idea de usar un combobox, pero podrias agregar los items con un for loop en vez de linea por linea.

Y tambien cambia el nombre de la variable, por que ya no es un line edit.